app/src/main/java/space/taran/arknavigator/mvp/presenter/adapter/ResourcesGridPresenter.kt

```diff
@@ -54,7 +55,12 @@ class ResourcesGridPresenter(
         val resource = selection[view.position()]
         Log.d(RESOURCES_SCREEN, "binding view for resource ${resource.id}")
 
-        val path = index.getPath(resource.id)
+        val path = try {
+            index.getPath(resource.id)
+        } catch (e: AssertionError) {
```

Comment:
I will remove this try-catch. This is the solution for #109. I thought it came from a refactoring.

Reply:
`AssertionError` is only an indicator of deeper problem (existing reference to removed resource). We should allow assertions to fail in debug mode (actually maybe even in production, since sometimes better to crash than destroy user's data).